test/JDBC/input/bbf_sysadmin_login_user-vu-verify.mix

```diff
@@ -0,0 +1,47 @@
+-- tsql
+-- run as sysadmin
+use bbf_sysadmin_login_user_db
+go
+create user bbf_sysadmin_login_user
+go
+
+-- allow when escape hatch enabled
+exec sp_babelfish_configure 'escape_hatch_allow_create_user_for_sysadmin', 'ignore'
+go
+create user bbf_sysadmin_login_user
+go
+exec sp_babelfish_configure 'escape_hatch_allow_create_user_for_sysadmin', 'strict'
+go
```

Comment:
Naming of EH seems a bit in contrast of what it is doing. Let's check with Stefan & Alex. My suggestion is to have 'escape_hatch_block_create_user_for_sysadmin' -- Since it is getting enabled when we ignore them.